app-admin/yadm/yadm-3.1.1.ebuild

```diff
@@ -44,19 +53,7 @@ src_compile() {
 }
 
 src_test() {
-	# test_encryption: needs write access to /tmp
-	# test_alt, test_compat_jinja, test_unit_template_j2: needs envtpl
-	# test_syntax: needs =dev-util/yamllint-1.17.0 (not packaged)
-	# test_compat_alt: known broken, tests deprecated features
-	# test_compat_jinja: needs envtpl
-	rm -v test/test_encryption.py || die
-	rm -v test/test_alt.py || die
-	rm -v test/test_compat_jinja.py || die
-	rm -v test/test_unit_template_j2.py || die
-	rm -v test/test_syntax.py || die
-	rm -v test/test_compat_alt.py || die
-
-	pytest || die "Testsuite failed under ${EPYTHON}"
+	epytest
```

Comment:
you'll need to inherit `python-utils-r1` to use that
also you need to define `EPYTEST_DESELECT` or `EPYTEST_IGNORE` ( both are arrays of paths)

first one passes `--deselect`
second obviously `--ignore`

try
```bash 
local EPYTEST_IGNORE=(
    test/test_encryption.py
   ...
)

epytest
```

idk what this package needs, but 1 of 2 should work.


btw. most eclasses have manpages
you can install `app-doc/eclass-manpages`
and check `man python-utils-r1.eclass`
really useful.

Reply:
Thank you.

Reply:
A bit unrelated, but your e-mail has typo in cargo.eclass man page - `@genotoo.org` :D